--- a/.vite/build/workspace-root-drop-handler-Cr2t_88z.js
+++ b/.vite/build/workspace-root-drop-handler-Cr2t_88z.js
@@ -35862,6 +35862,7 @@
     i = e.y(n.version);
   return (
     e.r({
+      enabled: !1,
       dsn: e.v,
       environment: PE.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -57034,6 +57034,7 @@
   let n = hL(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (XA({
+    enabled: !1,
     dsn: iB,
     environment: e.buildFlavor,
     release: aB(n.version),
